src/main/java/marquez/service/DatasetService.java

```diff
@@ -63,12 +110,20 @@ public Dataset create(
     }
   }
 
-  public Optional<Dataset> get(@NonNull DatasetUrn urn) throws MarquezServiceException {
+  public boolean exists(@NonNull DatasetUrn datasetUrn) throws MarquezServiceException {
```

Comment:
nit: why not just call it `urn`?

Reply:
Originally, as I was defining DatasetService.create()`, I had explicitly listed all params instead of passing in a `Dataset` object. So, `urn` became `datasetUrn`, but makes sense now to shorten it. Good catch.